setup.cfg

```diff
@@ -1,2 +1,6 @@
 [metadata]
 license_file = LICENSE
+
+[flake8]
```

Comment:
I see. I'm not familiar with flake8, but with pylint we had to ignore a lot of errors and warnings https://github.com/locationtech-labs/geopyspark/blob/master/pylintrc#L7 So anything we ignore for pylint will also need to be ignored for flake8 as well.